controllers/util/collection_util.go

```diff
@@ -85,6 +85,59 @@ func CreateCollecttionAlias(cloud string, alias string, aliasType string, collec
 
 }
 
+// CreateShard to request the creation of a shard to be created for a given collection
+func CreateShard(cloud string, collection string, shard string, createNodeSet []string, namespace string) (success bool, err error) {
+	queryParams := url.Values{}
+	NodeSetArray := strings.Join(createNodeSet, ",")
+	queryParams.Add("action", "CREATESHARD")
+	queryParams.Add("shard", shard)
+	queryParams.Add("collection", collection)
+	queryParams.Add("createNodeSet", NodeSetArray)
+
+	resp := &SolrAsyncResponse{}
+
+	log.Info("Calling to create shard", "namespace", namespace, "cloud", cloud, "shard", shard, "collections", collection, "createNodeSet", createNodeSet)
```

Comment:
"collections" plural, should be singular?

Reply:
yep, good looking out!